rancher-shell-0.4.yaml

```diff
@@ -0,0 +1,73 @@
+package:
+  name: rancher-shell-0.4
+  description: kubectl and helm installer image for Rancher
+  epoch: 0
+  version: 0.4.1
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - bash
+      - bash-completion
+      - busybox
+      - curl
+      - gnutar
+      - gzip
+      - helm
+      - jq
+      - k9s
+      - kubectl
```

Comment:
we likely need `kubectl-bash-completion` in here as well but we don't have provides for that as of now. :(

Reply:
```diff
 $ git diff
diff --git a/kubernetes-1.33.yaml b/kubernetes-1.33.yaml
index 3dd688ebd..f35a9e301 100644
--- a/kubernetes-1.33.yaml
+++ b/kubernetes-1.33.yaml
@@ -87,6 +87,8 @@ subpackages:

   - name: kubectl-bash-completion-${{vars.kubernetes-version}}
     dependencies:
+      provides:
+        - kubectl-bash-completion=${{package.full-version}}
       runtime:
         - kubectl-${{vars.kubernetes-version}}
     pipeline:
```
we need this change before including `kubectl-bash-completion` directly here.